services/common/src/main/java/org/collectionspace/services/common/relation/nuxeo/RelationConstants.java

```diff
@@ -54,6 +54,6 @@ public class RelationConstants {
     public final static String TYPE_SCHEMA_NAME = COMMON_SCHEMA_NAME;
     public final static String TYPE_FIELD_NAME = IRelationsManager.RELATIONSHIP_TYPE;
     
-    public final static String AFFECTS_TYPE = RelationshipType.AFFECTS.toString();
-    public final static String BROADER_TYPE = RelationshipType.HAS_BROADER.toString();
+    public final static String AFFECTS_TYPE = RelationshipType.AFFECTS.toString().toLowerCase();
+    public final static String BROADER_TYPE = RelationshipType.HAS_BROADER.toString().toLowerCase();
```

Comment:
These smell funny. Are we sure about this? It would make sense to me if they were `RelationshipType.AFFECTS.value()` and `RelationshipType.HAS_BROADER.value()`, but `toLowerCase()` seems arbitrary. It looks like these constants are only used by a botgarden listener and a botgarden batch job, therefore only by UCB, so if things are working as you expect, @remillet, I guess its fine? But are you really sure you want this?

Reply:
Sure, RelationshipType.AFFECTS.value() should work.

Looks like toString() was returning "AFFECTS" and value() returns "affects".  Lots of places in the code (including parts of the UI?) assume lowercase "affects".